src/angular/3a-pipes/3a-pipes.md

```diff
@@ -20,8 +20,14 @@ In this exercise, we will fix the path of the thumbnail images in **src/app/rest
 Currently the path is written out like:
 
 ```html
-<img src="{{ restaurant.images.thumbnail }}" width="100" height="100" />
+<img
+  alt=""
```

Comment:
@chasenlehara I think empty alt tags dont add any value

Reply:
They do! Without an `alt` attribute, the file name will be read by screenreaders. For these images I think it’s ok if we treat them as decorative (not conveying important content), so `alt=""` will make screenreaders skip over them.